pennylane/measurements/sample.py

```diff
@@ -55,10 +55,37 @@ def sample(op: Optional[Union[Operator, MeasurementValue]] = None, wires=None) -
     .. note::
 
         QNodes that return samples cannot, in general, be differentiated, since the derivative
-        with respect to a sample --- a stochastic process --- is ill-defined. The one exception
-        is if the QNode uses the parameter-shift method (``diff_method="parameter-shift"``), in
-        which case ``qml.sample(obs)`` is interpreted as a single-shot expectation value of the
-        observable ``obs``.
+        with respect to a sample --- a stochastic process --- is ill-defined. An alternative
+        approach would be to use single-shot expectation values. For example, instead of this:
```

Comment:
Is this specific to parameter shift or does it apply to other diff methods as well?

Reply:
not necessarily, but parameter-shift is the only one we've got that works with finite shots other than `finite-diff`, and the results from that are just... shocking. don't use finite-diff : )